File: app/src/main/java/com/cliniva/enventory/viewholder/UnpaidHolder.java
package com.cliniva.enventory.viewholder;

import android.databinding.ViewDataBinding;
import android.support.annotation.NonNull;

import com.cliniva.enventory.adapter.base.BaseRecyclerClickListener;
import com.cliniva.enventory.adapter.base.BaseRecyclerViewHolder;
import com.cliniva.enventory.databinding.ItemUnpaidBinding;
import com.cliniva.enventory.model.Transaction;

public class UnpaidHolder extends BaseRecyclerViewHolder<Transaction, BaseRecyclerClickListener<Transaction>> {

    private ItemUnpaidBinding binding;

    public UnpaidHolder(@NonNull ViewDataBinding itemView) {
        super(itemView);
    }

    @Override
    public void onBindView(Transaction item, int position, BaseRecyclerClickListener<Transaction> listener) {
        enableItemViewClick(item, listener);
        binding.setUnpaid(item);
    }
}

File: app/src/main/java/com/amirsons/inventory/ui/recyclerview/viewholder/UnpaidHolder.kt
package com.amirsons.inventory.ui.recyclerview.viewholder

import android.view.View

import com.amirsons.inventory.ui.recyclerview.base.BaseRecyclerClickListener
import com.amirsons.inventory.ui.recyclerview.base.BaseRecyclerViewHolder
import com.amirsons.inventory.datamanager.model.Transaction

class UnpaidHolder(itemView: View) : BaseRecyclerViewHolder<Transaction, BaseRecyclerClickListener<Transaction>>(itemView) {

    override fun onBindView(item: Transaction, position: Int, listener: BaseRecyclerClickListener<Transaction>?) {
        enableItemViewClick(item, listener)
    }
}
